hooks/useWeatherStationTimeseries.ts

```diff
@@ -0,0 +1,98 @@
+import {useContext} from 'react';
+
+import {useQuery, useQueryClient} from 'react-query';
+
+import {ClientContext, ClientProps} from 'clientContext';
+import {ApiError, OpenAPI, StationMetadata, TimeseriesDataService} from 'types/generated/snowbound';
+import {AvalancheCenterID} from 'types/nationalAvalancheCenter';
+import AvalancheCenterMetadata from 'hooks/useAvalancheCenterMetadata';
+import {toSnowboundStringUTC} from 'utils/date';
+
+type Source = 'nwac' | 'snotel' | 'mesowest';
+
+interface Props {
+  center: AvalancheCenterID;
+  stids: string[];
+  sources: Source[];
+  startDate: Date;
+  endDate: Date;
+}
+
+type Variable =
```

Comment:
How is this different from `types/snowbound.ts`?

Reply:
well, it's different because you made that one and I made this one?

Reply:
just kidding, I didn't see the other one

Reply:
OK, just hoping to save you time here ... that was why I had already implemented the entire API thing and had pages showing data ...

Reply:
Ah. I was trying to lean on the OpenAPI definitions and only fill in gaps around those, which is why I ended up duplicating work for these types 😬 

Are the API shapes identical, or subtly different? I haven't dug into them enough to see.